### PR TITLE
Updating number of replica for existing indices while deploying openshift-logging

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1740669

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
LGTM

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -1223,6 +1226,22 @@ the new `openshift_logging_es_cluster_size` value is higher than the current num
...
 
+[[scaling-elasticsearch-replicas]]
+===== Changing the Number of Elasticsearch Replicas =====
+
+You can change the number of Elasticsearch replicas by editing the `openshift_logging_es_number_of_replicas` value
+in the inventory host file and re-running the logging playbook as described previously.
+
+The changes apply only to the new indices. Existing indices continue to use the previous number of replicas. 
+For example, if you change the number of indices from 3 to 2, your cluster will use 2 replicas for new indices 
+and 3 replicas for already existing indices.
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc:1237]: ~already~

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -1223,6 +1226,22 @@ the new `openshift_logging_es_cluster_size` value is higher than the current num
...
+===== Changing the Number of Elasticsearch Replicas =====
+
+You can change the number of Elasticsearch replicas by editing the `openshift_logging_es_number_of_replicas` value
+in the inventory host file and re-running the logging playbook as described previously.
+
+The changes apply only to the new indices. Existing indices continue to use the previous number of replicas. 
+For example, if you change the number of indices from 3 to 2, your cluster will use 2 replicas for new indices 
+and 3 replicas for already existing indices. 
+
+You can modify the replica count for the existing indices by executing a command such as:
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc:1239]: s/executing/running

I'd say "by running the following command" and use an annotation to explain which parameters you're expecting them to modify.

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but it LGTM.

**Pull request merged**